src/components/ebay-textbox/README.md

```diff
@@ -0,0 +1,44 @@
+# ebay-textbox
+
+## ebay-textbox Usage
+
+Default input textbox:
+
+```marko
+<ebay-textbox />
```

Comment:
Keep this most basic example in the `README`, but remove the other ones. We want to avoid repeating documentation that is already covered in the `Attributes` section below, and the `examples/` directory.